Dockerfile-fpm

```diff
@@ -49,6 +53,10 @@ VOLUME [ "/var/run" ]
 ## FPM-DEV STAGE ##
 FROM fpm as fpm-dev
 
+# Override the configuration to symlink to the recommended development configuration
+# ref: https://github.com/docker-library/docs/tree/master/php#configuration
+RUN ln -sf $PHP_INI_DIR/php.ini-development $PHP_INI_DIR/php.ini
```

Comment:
This will only take effect for people download the `-dev` images, we want to make this part of our `docker-php-dev-mode config` script, thus it's consistently installed whenever someone enables dev mode in their setups

Same applies for the `cli` image

And documentation can be updated accordingly